Agents/DataBridgeAgent/README.md

```diff
@@ -0,0 +1,129 @@
+```
+docker-compose up -d
+```
+
+#### 2.4 **STACK ENVIRONMENT**
+- Follow the following instructions to deploy the agent within a stack.
+  1) If the image is not available on the repository, build the image with the following tag at the `<root>` directory. Please change version number for your needs.
+```
+docker build -t data-bridge-agent:versionNo .
+```
+  2) Add the `<root>/docker/data-bridge-agent.json` to the [`stack-manager/inputs/config/services`](https://github.com/cambridge-cares/TheWorldAvatar/blob/main/Deploy/stacks/dynamic/stack-manager/inputs/config/services) directory
+     - Please ensure the version numbers are targeted at the right image. If you are building it, please update the version number accordingly.
+  3) Modify the absolute path of the agent's `config` folder to your absolute file path
+     - For Windows users using WSL on Docker, the file path should start with `/mnt/c/`, which is equivalent to `C://`
+  4) Include this agent service into the stack configuration file at `stack-manager/inputs/config/<STACK-NAME>.json`
+     - Read more in the [Stack Configuration](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Deploy/stacks/dynamic/stack-manager) section
+  5) Start the stack as per normal
+
+If the agent is successfully started, the endpoint at `http://localhost:3838/data-bridge-agent/status` should return the following message.
+```
+{"Result":"Agent is ready to receive requests."}
+```
+
+### 3. Endpoint
+When successfully built, the agent will be running at `port 3055`. The base url will be `http://localhost:3055/data-bridge-agent`.
+There are currently four routes available:
+
+1. `<base>/status` route:
+   - Returns the current status of the agent through an HTTP `GET` request.
+2. `<base>/sparql` route:
+    - Execute the agent's task through an HTTP `POST` request. This route will transfer data between the specified source and target endpoints.
+    - The request will require the following parameters:
+      - `source`: The source SPARQL endpoint containing the triples to be transferred 
+      - `target`: The target SPARQL endpoint intended to store the transferred triples
+    - Sample SPARQL endpoints for Blazegraph are [listed here](#4-sample-blazegraph-endpoints)
+    - **WARNING**: The transfer of triples may fail for large (>1 million) numbers of triples as it is dependent on the available RAM. The [JPS Base library's cloning tool](https://github.com/cambridge-cares/TheWorldAvatar/blob/main/JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/tools/cloning/CloningTool.java) can be used in that situation.
+    - A sample `POST` request using curl on a CLI:
+```
+curl -X POST --header "Content-Type: application/json" -d "{
+    'source':'http://user:pass@ipaddress:port/blazegraph/namespace/kb/sparql',
+    'target': 'http://user:pass@stackName-blazegraph:8080/blazegraph/namespace/kb/sparql'
+}" localhost:3055/data-bridge-agent/sparql 
```

Comment:
I would recommend swapping single and double quotes in this `curl` example, because, as we have learnt the hard way recently, with single quotes it is not valid JSON! Many pieces of software don't seem to care, but some do!